Duplicati/Library/Backend/Telegram/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net471" />
+  <package id="System.Net.Requests" version="4.3.0" targetFramework="net471" />
+  <package id="TLSharp" version="0.1.0.573" targetFramework="net471" />
```

Comment:
We should add the license information, etc. for `TLSharp` to the `Duplicati.License` project.